2025/2025-09-08-sparklend-mainnet-withdraw-usds-dai-reserves.md

```diff
@@ -0,0 +1,57 @@
+---
+title: SparkLend Mainnet - Withdraw USDS and DAI Reserves from SparkLend - September 8, 2025
+summary: Signal your support or opposition to withdrawing USDS and DAI reserves from SparkLend to the Spark Proxy on mainnet.
+discussion_link: https://forum.sky.money/t/september-18-2025-proposed-changes-to-spark-for-upcoming-spell/27153
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2025-09-08T16:00:00
+end_date: 2025-09-11T16:00:00
+---
+
+# Poll: SparkLend Mainnet - Withdraw USDS and DAI Reserves from SparkLend - September 8, 2025
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Stability Facilitators. This Governance [Poll](https://sky-atlas.powerhouse.io/A.1.10.1_Operational_Weekly_Cycle/b189fa17-57a9-4d4e-9780-0ce4efd94211|0db30308) will be active for three days beginning on Monday, September 8 at 16:00 UTC.
+
+**This is a binary vote.**
+
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to transferring all accumulated DAI and USDS reserves from the SparkLend Collector to the Spark Proxy on mainnet with the following details:
```

Comment:
We should use the same word for the same action. Previously we used "withdrawing"

Reply:
Also, they will be deposited as USDS